.buildkite/scripts/test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -euo pipefail
+
+echo "--- Pre install"
+source .buildkite/scripts/pre-install-command.sh
+add_bin_path
+with_go_junit_report
+
+echo "--- Test"
+set +e
+go test -v ./... > tests-report.txt
+exit_code=$?
+set -e
+
+# Buildkite collapse logs under --- symbols
+# need to change --- to anything else or switch off collapsing (note: not available at the moment of this commit)
+awk '{gsub("---", "----"); print }' tests-report.txt
+
+# Create Junit report for junit annotation plugin
+go-junit-report > junit-report.xml < tests-report.txt
```

Comment:
I was discussing this with @pazone today, I think it would be better to move the `go-junit-report` into its own step in the pipeline.

Reply:
it will run in a separate container then.... but i changed it